client/curriculum/curriculum.py

```diff
@@ -0,0 +1,43 @@
+from random import random
+import yaml
+
+
+def curriculum_learning(obj, file):
+    """Curriculum training for better generalization, currently implemented in
```

Comment:
Disregard this comment if the function is going to change significantly for the final implementation, but the `file` and `obj` arguments should be better documented and defined.

Reply:
I've added some comments. Please let me know if you think this is enough or if I should elaborate further! Thanks for the great feedback.